css/css-lists/counter-reset-reversed-not-list-item.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Lists: counter-reset reversed(not-list-item)</title>
+<link rel="help" href="https://drafts.csswg.org/css-lists/#counter-reset">
+<link rel="match" href="counter-7-ref.html">
+<style>
+  span { counter-increment: not-list-item -2; }
+  .reset-reversed { counter-reset: reversed(not-list-item) }
+  .result::before { content: counter(not-list-item) }
+</style>
+<p>You should see the number 7 below.</p>
```

Comment:
It looks like you're expecting the reversed counter to start on 13 here, meaning you're leaning towards "count the number of elements that apply an increment of any kind to the counter" for the impl.

If we're already dynamically counting things that actively increment the counter, wouldn't we want to do it "correctly" and take the actual increment into account, like I suggest above?

Reply:
This is now fixed.